### DISCUSSION TITLE
Steps to migrate from experimental M1 2.7 to 3.0

### DISCUSSION THREAD

**Discussion by kellertuer** [Homebrew/discussions#727] [Getting started]
Great to see, that with 3.0 home-brew has officially the M1 in its features.

I already installed 2.7 in the M1 scheme (i.e. without Rosetta) but in `opt/loca/` instead of `var/local` (though I never installed the Rosetta one there). Should I switch this folder when migrating?

**Comment by carlocab** [comment]
See #707, #417, #437, #545, #357.

The short answer is that you'll want to use the installer to install things into `/opt/homebrew` rather than moving things manually.

**Reply by kellertuer** [reply]
Well, but I think I never did the Rosetta2 thing, or am I mistaken? Maybe I remember that wrongly, I have to check. And then try to understand how the installer works.

_Edit_: Especially I am not running my terminal with rosetta.

**Reply by kellertuer** [reply]
Ah, now I see; the prefixes did not change, so my install will just stay in `opt/homebrew` then. That's where my mistake was, I thought with the official switch to support M1, it might also move to `usr/...` which it does not. Then my native variant is already in the right place (as I said, I never installed a rosetta variant).
